### ISSUE TITLE
Form validation accessibility

### ISSUE THREAD

**Issue by mmazanec22** [ctoec/ecis-experimental#511]
- Invalid input should be marked with aria-invalid
- Use "required" attribute for required fields
- “Move the focus to the first form control that has the error or move the focus to the common error message.” [see Deque guide on error messages](https://www.deque.com/blog/anatomy-of-accessible-forms-error-messages/)
- Revisit datepicker-- currently can't use aria-describedby-- research to ensure this is an issue and either create an issue for the library or figure out an alternative

**Comment by mmazanec22** [comment]
Also: fields that should be numbers should be type="number", can validate length of text for things like state, etc

**Comment by mmazanec22** [comment]
Closed by #619
